sass/directives/04_utilities/_gating.scss

```diff
@@ -0,0 +1,25 @@
+@mixin card-group-gate {
```

Comment:
I will move these back to employer. However they will be added to a sheet that is agnostic of small-biz and why-cb. These cards will most likely render out the cards found through the site so they're not entirely specific to any page. 

That was the original reason I added them to our style base.

Reply:
So this mixin was split, with  `_gating.scss` being removed. The styles specific to card-group were kept in `_card_group.scss` since they're very generic and yes, if a 3rd party wants to include the logo w/ hover, it would need to be disabled for the iPhone and iPad due to the doubletap bug.

The other part dealing with the `small-biz` content (actually it was changed to `why-cb` was moved back to employer for the reasons you stated.

Reply:
👍 and sorry, this was a bit blunt, I was in a rush to submit this this morning